### ISSUE TITLE
replacer and technic cable crash

### ISSUE THREAD

**Issue by OgelGames** [pandorabox-io/pandorabox.io#465]
```
2020-03-08 00:21:12: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'vacuum' in callback item_OnUse(): ...//worldmods/technic/technic/machines/register/cables.lua:231: table index is nil
2020-03-08 00:21:12: ERROR[Main]: stack traceback:
2020-03-08 00:21:12: ERROR[Main]: 	...//worldmods/technic/technic/machines/register/cables.lua:231: in function 'on_place'
2020-03-08 00:21:12: ERROR[Main]: 	/data/world//worldmods/replacer/replacer.lua:190: in function 'replace_single_node'
2020-03-08 00:21:12: ERROR[Main]: 	/data/world//worldmods/replacer/replacer.lua:279: in function </data/world//worldmods/replacer/replacer.lua:224>
```
```
2020-03-08 00:31:21: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'vacuum' in callback item_OnUse(): ...//worldmods/technic/technic/machines/register/cables.lua:231: table index is nil
2020-03-08 00:31:21: ERROR[Main]: stack traceback:
2020-03-08 00:31:21: ERROR[Main]: 	...//worldmods/technic/technic/machines/register/cables.lua:231: in function 'on_place'
2020-03-08 00:31:21: ERROR[Main]: 	/data/world//worldmods/replacer/replacer.lua:190: in function 'replace_single_node'
2020-03-08 00:31:21: ERROR[Main]: 	/data/world//worldmods/replacer/replacer.lua:279: in function </data/world//worldmods/replacer/replacer.lua:224>
```
https://github.com/mt-mods/technic/blob/master/technic/machines/register/cables.lua#L231

Seems like an attempt to place `technic` cables with a replacer again. 

Not sure if `vacuum` actually has anything to do with it...

**Comment by coil0** [comment]
Fixed by minetest-mods/technic#534.

**Comment by BuckarooBanzay** [comment]
thanks @coil0 :+1: i cherry-picked the commit into the `mt-mods/technic` repo directly and updated the pandorabox-mods repo (otherwise we wait a few weeks until that trickles down from the upstream repo...)
